sql-native/src/main/java/org/ballerinalang/sql/utils/Utils.java

```diff
@@ -129,7 +129,7 @@ static void closeResources(TransactionResourceManager trxResourceManager, Result
         }
     }
 
-    static String getSqlQuery(BObject paramString) {
+    private static String getSqlQuery(BObject paramString) {
         BArray stringsArray = paramString.getArrayValue(Constants.ParameterizedQueryFields.STRINGS);
```

Comment:
Change the access modifiers of methods that need in parameter processors to public